internal/http/parser/invalid_errors.yaml

```diff
@@ -0,0 +1,55 @@
+# Protocol Errors
+
+- input: "HTTP/3.0 200 OK"
+  error: |
+    Received: "HTTP/3.0 200 OK"
+                    ^ error
+    Error: Expected HTTP-version 1.1, Received: 3.0
+
+- input: "HTTP/1.0_200_OK"
+  error: |
+    Received: "HTTP/1.0_200_OK"
+                    ^ error
+    Error: Expected HTTP-version 1.1, Received: 1.0
+
+- input: "HTPP/1.0 200 OK"
+  error: |
+    Received: "HTPP/1.0 200 OK"
+               ^ error
+    Error: Expected HTTP/1.1, Received: HTPP/1.0
+
+- input: "HTTP|1.0 200 OK"
+  error: |
+    Received: "HTTP|1.0 200 OK"
+               ^ error
+    Error: Expected HTTP/1.1, Received: HTTP|1.0
+
+- input: "HTTP//1.0 200 OK"
+  error: |
+    Received: "HTTP//1.0 200 OK"
+                    ^ error
+    Error: Expected HTTP-version 1.1, Received: /1.
+
+- input: "HTTP/1 200 OK"
+  error: |
+    Received: "HTTP/1 200 OK"
+                    ^ error
+    Error: Expected HTTP-version 1.1, Received: 1 2
```

Comment:
In cases where the `HTTP/` prefix is there, we are trying to tell the users that the version is wrong. Works correctly most of the times, but in some cases can give weird errors. 
Should we remove this ? 
And only say `Expected HTTP/1.1, Received: foo`

Reply:
> And only say Expected HTTP/1.1, Received: foo

Yes

Reply:
(Add quotes to the values)